pkg/e2e/state/prometheus.go

```diff
@@ -27,7 +27,8 @@ var _ = ginkgo.Describe("[Suite: e2e] Cluster state", func() {
 		// this command is has specific code to capture and suppress an exit code of
 		// 1 as tar 1.26 will exit 1 if files change while the tar is running, as is
 		// common for a running prometheus instance
-		cmd := promCollectCmd + " >" + runner.DefaultRunner.OutputDir + "/prometheus.tar.gz\" ; err=$? ; if (( $err != 1 )) ; then exit $err ; fi"
+		cmd := promCollectCmd + "\" > " + runner.DefaultRunner.OutputDir + "/prometheus.tar.gz; err=$? ; if (( $err != 0 )) ; then exit $err ; fi"
```

Comment:
Let's move the \" into the promCollectCmd rather than here.

Reply:
Agree